model/task/task.go

```diff
@@ -1173,6 +1173,33 @@ func (t *Task) MergeNewTestResults() error {
 	return nil
 }
 
+func MergeTestResultsBulk(tasks []Task, query *db.Q) ([]Task, error) {
```

Comment:
Explain the behavior of this function in a comment, since it's potentially confusing that you can pass nil and get some default behavior or pass a query.